gaia-sdk-java/gaia-sdk-api/src/main/kotlin/gaia/sdk/api/skill/SkillSpec.kt

```diff
@@ -14,27 +14,33 @@ data class UnprovisionedSkillSpec(val owner: String,
                                   val name: String,
                                   val version: String,
                                   val options: ProvisioningOptions.() -> Unit = {}) : ISkillSpec {
-    override fun toURL() = "skill://$owner/$name/$version"
+    override fun toUri() = "skill://$owner/$name/$version"
+
+    // TODO
```

Comment:
Was hat es mit dem TODO aufsich?

Reply:
Das ist dann fuer die "on the fly" skill provision Erstellung, die wir noch umsetzen sollen